management/src/test/java/org/ehcache/management/cluster/ClusteringManagementServiceTest.java

```diff
@@ -241,7 +242,7 @@ public void test_stats_collection() throws Exception {
     assertThat(stats[0].getContext().get("cacheName"), equalTo("cache-1"));
     assertThat(stats[0].getStatistic(Counter.class, "GetCounter").getValue(), equalTo(4L));
     assertThat(stats[0].getContext().get("cacheName"), equalTo("cache-1"));
-    assertThat(stats[0].getStatistic(CounterHistory.class, "AllCacheGetCount").getValue().length, equalTo(1));
+    assertThat(stats[0].getStatistic(CounterHistory.class, "AllCacheGetCount").getValue().length, greaterThanOrEqualTo(1));
```

Comment:
@mathieucarbou I guess if it is always the same value, then the assertion could make sure all values are 4 - or at least one value is 4. The comment from @chrisdennis was that the fixed line felt weird compared to the check following it as the race could happen in different ways.